src/api/procedures/__tests__/transferTickerOwnership.ts

```diff
@@ -89,16 +99,29 @@ describe('transferTickerOwnership procedure', () => {
 
   beforeEach(() => {
     transaction = dsMockUtils.createTxMock('identity', 'addAuthorization');
-
     mockContext = dsMockUtils.getContextInstance();
+    target = entityMockUtils.getIdentityInstance({ did: args.target as string });
```

Comment:
```suggestion
    target = entityMockUtils.getIdentityInstance({ did });
```

Could this work? it looks like `did` is constant `someDid`. Seems like `as string` is repeated a few times for `args.target`